### DISCUSSION TITLE
Library of files view

### DISCUSSION THREAD

**Discussion by MariaGanan** [2sic/2sxc#3335] [General] [answered]
Hi,
I have a question for the library of files.
This is the way it should be shown here? 
![imagen](https://github.com/2sic/2sxc/assets/139103286/a3102b8d-284f-45d8-a881-326ae32d14e7)

There is no Icon and there is only 5 characters, maybe I'm doing something wrong here or it is this way?

Thanks.

**Comment by iJungleboy** [comment]
I believe you created a subfolder... That's why you only see 1 tile.

**Reply by MariaGanan** [reply]
Hi, 
Here I have 4 files
![imagen](https://github.com/2sic/2sxc/assets/139103286/646874f0-fb73-4c04-ae22-2d52d814a47d)
Is this the view for documents? there is no icon or something?

Thanks

**Reply by iJungleboy** [reply, marked as answer]
images get a preview.
I assume they are not images.
Not sure about other file types.

**Reply by MariaGanan** [reply]
Yes, they are not images. 
Ok, thanks, I wanted to be sure that I was not doing anything wrong.
